mne/evoked.py

```diff
@@ -402,10 +402,11 @@ def plot(self, picks=None, exclude='bads', unit=True, show=True, ylim=None,
             channel traces will not be shown.
         window_title : str | None
             The title to put at the top of the figure window.
-        spatial_colors : bool
-            Color code lines by mapping physical sensor coordinates into color
-            values. Spatially similar channels will have similar colors.
-            Bad channels will be dotted.
+        spatial_colors : None | 'head' | 'skirt' | 'notopo'
```

Comment:
;) I'd say keep it simple. For advanced users, they can access the `axis` for the legend and modify it to suit their needs. Or internally have an `auto` mode which selects `head` or `skirt`.